.gitignore

```diff
@@ -9,6 +9,9 @@
 /tmp/
 /*.gem
 /vendor
+bin/*
```

Comment:
`/bin` already excluded above. Exclusion should start with `/`, else it excludes every folder named `bin` anywhere in project.

Reply:
I added the `/` to `bin`.

I also reinstated `bin/setup` and `bin/console` and put exceptions in gitignore for them. Unlike the other files in `/bin`, they are not generated by the bundle command but rather they are files created when the gem is created.

They are also referenced in the Development section of README.md